dotnet/src/SemanticKernel.Core/Kernel.cs

```diff
@@ -194,6 +194,10 @@ public Task<SKContext> RunAsync(CancellationToken cancellationToken, params ISKF
     public Task<SKContext> RunAsync(string input, CancellationToken cancellationToken, params ISKFunction[] pipeline)
         => this.RunAsync(new ContextVariables(input), cancellationToken, pipeline);
 
+    /// <inheritdoc/>
```

Comment:
Trying to reduce these overloads. Why is this one needed? I'd remove it from both places

Reply:
You are right, using just one impl. No more overloads.